nc/source/physics/dynamics/Solver.cpp

```diff
@@ -316,19 +316,19 @@ namespace
         // Get anchor world space positions
         XMVECTOR pA, pB;
         {
-            auto* transformA = registry->Get<Transform>(joint.entityA);
-            auto positionA = transformA->GetPosition();
-            auto rotationA = transformA->GetRotation();
-            auto rotMatrix = XMMatrixRotationQuaternion(XMLoadQuaternion(&rotationA));
+            auto* transform = registry->Get<Transform>(joint.entityA);
+            auto position = transform->GetPosition();
+            auto rotation = transform->GetRotation();
+            auto rotMatrix = XMMatrixRotationQuaternion(XMLoadQuaternion(&rotation));
             joint.rA = XMVector3Transform(joint.anchorA, rotMatrix);
-            pA = XMLoadVector3(&positionA) + joint.rA;
+            pA = XMLoadVector3(&position) + joint.rA;
 
-            auto* transformB = registry->Get<Transform>(joint.entityB);
-            auto positionB = transformB->GetPosition();
-            auto rotationB = transformB->GetRotation();
-            auto rotMatrix = XMMatrixRotationQuaternion(XMLoadQuaternion(&rotationB));
+            transform = registry->Get<Transform>(joint.entityB);
+            position = transform->GetPosition();
+            rotation = transform->GetRotation();
+            rotMatrix = XMMatrixRotationQuaternion(XMLoadQuaternion(&rotation));
```

Comment:
ha - this double declare of rotMatrix popped up on my merge this morning. I see its fixed now.